### PR TITLE
updated readme descriptions and links

### PR DESCRIPTION
**Pull request by jennamk14**
Updated descriptions of links to include which ones redirect to HF
Updated links to scripts within the repo

### CODE REVIEW

**Comment by jennamk14** [comment]
@egrace479 question on readme - when referencing terminal commands to run the scripts, for example line 79 `detector2cvat --video path_to_videos --save path_to_save` does the detector2cvat need to have a .py at the end? or is this not needed for scripts part of the pyproject? thanks!

File: README.md [hunk 1]
```diff
@@ -56,15 +56,16 @@ frame-by-frame behavior annotation.
 
 #### To create mini-scenes, we first must perform the detection step, by drawing bounding boxes around each animal in frame. 
 
-See [data/mini_scenes](https://huggingface.co/imageomics/x3d-kabr-kinetics/tree/main/data/mini_scenes) for example mini-scenes.
+See [data/mini_scenes](https://huggingface.co/imageomics/x3d-kabr-kinetics/tree/main/data/mini_scenes) in HuggingFace for example mini-scenes.
```
Comment (egrace479) [inline README.md:59]:
```suggestion
See [data/mini_scenes](https://huggingface.co/imageomics/x3d-kabr-kinetics/tree/main/data/mini_scenes) on Hugging Face for example mini-scenes.
```

**Review by egrace479 — COMMENTED** [review]
A few small changes.

As to your question @jennamk14, as a package it is a command that can be run now (so no `.py`, that would be executing the file).

File: README.md [hunk 2]
```diff
@@ -90,23 +91,23 @@ tracks_extractor --video path_to_videos --annotation path_to_annotations [--trac
 ```
 
 ## Step 3: Label mini-scenes with behavior 
-You can use the [KABR model](https://huggingface.co/imageomics/x3d-kabr-kinetics) to label the mini-scenes with behavior. See the [ethogram](ethogram) folder for the list of behaviors used to label the zebra videos.
+You can use the [KABR model](https://huggingface.co/imageomics/x3d-kabr-kinetics) in HuggingFace to label the mini-scenes with behavior. See the [ethogram](ethogram) folder for the list of behaviors used to label the zebra videos.
 
 
-To use the [KABR model](https://huggingface.co/imageomics/x3d-kabr-kinetics), download `checkpoint_epoch_00075.pyth.zip`, unzip `checkpoint_epoch_00075.pyth`, and install [SlowFast](https://github.com/facebookresearch/SlowFast). Then run [miniscene2behavior.py](miniscene2behavior.py).
+To use the [KABR model](https://huggingface.co/imageomics/x3d-kabr-kinetics), download `checkpoint_epoch_00075.pyth.zip`, unzip `checkpoint_epoch_00075.pyth` from HuggingFace, and install [SlowFast](https://github.com/facebookresearch/SlowFast). Then run [miniscene2behavior.py](miniscene2behavior.py).
```
[the remark this answers is not in the document]
Comment (egrace479) [reply README.md:97]: @jennamk14, I think this would be the only edit needed here.

**Review by egrace479 — APPROVED** [review]

**Pull request merged**